pkg/artifactcache/handler.go

```diff
@@ -296,7 +315,7 @@ func (h *Handler) commit(w http.ResponseWriter, r *http.Request, params httprout
 	}
 
 	cache.Complete = true
-	if err := h.db.Update(cache.ID, cache); err != nil {
+	if err := db.Update(cache.ID, cache); err != nil {
```

Comment:
<img width="673" alt="image" src="https://github.com/nektos/act/assets/9418365/d1f672ce-05ce-4209-9cda-6a67db1445ca">

`h.storage.Commit` is a time-consuming operation, maybe we cound close the db, and reopen it when `db.Update`